packages/story-editor/src/components/library/panes/media/local/mediaRecording/modal.js

```diff
@@ -64,6 +65,72 @@ function Modal({ isOpen, onClose }) {
   const [isImageCapture, setIsImageCapture] = useState(false);
   const [enableVideo, setEnableVideo] = useState(true);
   const [enableAudio, setEnableAudio] = useState(true);
+  const [videoInput, setVideoInput] = useState(null);
+  const [audioInput, setAudioInput] = useState(null);
+  const [mediaDevices, setMediaDevices] = useState([]);
+
+  const updateMediaDevices = useCallback(async () => {
+    try {
+      const devices = await navigator.mediaDevices.enumerateDevices();
+      setMediaDevices(
+        devices.filter((device) => device.kind !== 'audiooutput')
+      );
+    } catch (err) {
+      // eslint-disable-next-line no-console -- TODO: Figure out error handling.
```

Comment:
Is there an issue for this or just going to leave as a todo?

Reply:
Leaving as a todo for now since this depends on UX, which is still in flux.